aws-janitor/resources/route53.go

```diff
@@ -96,7 +96,9 @@ func route53ResourceRecordSetsForZone(opts Options, logger logrus.FieldLogger, s
 			}
 			logger.Warningf("%s: deleting %T: %s", o.ARN(), rrs, *rrs.Name)
 			if !opts.DryRun {
-				toDelete = append(toDelete, o)
+				if _, ok := opts.SkipResourceRecordSetTypes[*rrs.Type]; !ok{
+					toDelete = append(toDelete, o)
```

Comment:
Another thing we should keep in mind is that according to AWS tutorial:
https://docs.aws.amazon.com/cli/latest/reference/route53/change-resource-record-sets.html
Warning
To delete the resource record set that is associated with a traffic policy instance, use [DeleteTrafficPolicyInstance](https://docs.aws.amazon.com/Route53/latest/APIReference/API_DeleteTrafficPolicyInstance.html) . Amazon Route 53 will delete the resource record set automatically. If you delete the resource record set by using ChangeResourceRecordSets , Route 53 doesn't automatically delete the traffic policy instance, and you'll continue to be charged for it even though it's no longer in use.

I will add corresponding logic in follow up changes

Reply:
We can skip it for now if traffic policies are not being created.